File: ProcessorTest/app/src/main/java/com/example/processortest/MainActivity.java
package com.example.processortest;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

import com.example.annotation.BindView;
import com.example.api.InjectHelper;

public class MainActivity extends AppCompatActivity {
    @BindView(R.id.text_hello)
    TextView textView;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        InjectHelper.inject(this);
        textView.setText("xkf");
    }
}

File: HenCoderTest/class20/src/main/java/com/example/class20/MainActivity.kt
package com.example.class20

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import test

class MainActivity : AppCompatActivity() {
    
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        
        test()
    }
}
